src/App.css

```diff
@@ -1,38 +1,47 @@
-.App {
-  text-align: center;
+/*Universal CSS*/
+
+header {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #591902;
+}
+
+nav {
+  font: Arial, Helvetica, sans-serif;
+  color: #591902;
+}
+
+body {
```

Comment:
You can include the main font we're using in the body tag so it doesn't have to be defined in every element tag we use it in